src/scss/_vars.scss

```diff
@@ -114,3 +95,18 @@ $space: (
   xxl: 4.5rem,
   flush: 0
 );
+
+// ---------------------------------------- //
+// ----------------- GRID ----------------- //
+// ---------------------------------------- //
+$gutter-size: 20px;
```

Comment:
This is a new variable to support components like Grid and List that need to apply spacing between items. (This may or may not be replaced with spacing variables.)